reference/checkouts.v3.yml

```diff
@@ -420,12 +419,10 @@ paths:
                         product_id: 167
                         sku: ''
                         name: Able Brewing System
-                        url: >-
-                          http://store-url.mybigcommerce.com/all/able-brewing-system/
+                        url: 'http://store-url.mybigcommerce.com/all/able-brewing-system/'
                         quantity: 1
-                        taxable: true
-                        image_url: >-
-                          https://cdn8.bigcommerce.com/s-{store_hash}/products/167/images/398/ablebrewingsystem4_1024x1024__07155__93974.1534344516.330.500.jpg?c=2
+                        is_taxable: true
```

Comment:
Do all Checkouts V3 line_items' objects have the `is_taxable` field? Jira ticket mentions that the `gift_certificates` object has a `taxable` field instead of `is_taxable`.